src/main/js/controls/Bar/helpers/goalLineHelpers.js

```diff
@@ -139,12 +140,13 @@ const createRegion = (scale, config, regionGroupSVG, regionList, uniqueKey) => {
  * @param {object} scale - d3 scale taking into account the input parameters
  * @param {object} config - config object derived from input JSON
  * @param {object} regionGroupSVG - d3 object of region group svg
+ * @param {object} transition - gets transition based on pannig mode is enabled or not
  * @returns {undefined} - returns nothing
  */
-const translateRegion = (scale, config, regionGroupSVG) => {
+const translateRegion = (scale, config, regionGroupSVG, transition) => {
```

Comment:
This parameter is not necessary since you will have the required property `config.settingsDictionary.transition` within config.

Reply:
yes, I will remove it after adding settingDictionary in config